examples/src/main/java/ExampleUpdateVoiceCallFlow.java

```diff
@@ -0,0 +1,44 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.NotFoundException;
+import com.messagebird.exceptions.UnauthorizedException;
+
+public class ExampleCreateVoiceCallFlow {
+
+    public static void main(String[] args) {
+        if (args.length < 6) {
+            System.out.println("Please specify your access key and voice call flow arguments");
+            return;
+        }
+
+        //First create your service object
+        final MessageBirdService wsr = new MessageBirdServiceImpl(args[0]);
+
+        //Add the service to the client
+        final MessageBirdClient messageBirdClient = new MessageBirdClient(wsr);
+
+        final VoiceCallFlowRequest voiceCallFlow = new VoiceCallFlowRequest();
```

Comment:
"final VoiceCallFlowRequest voiceCallFlow " variable name has to be voiceCallFlowRequest because of rest of code